preinstall.js

```diff
@@ -66,7 +76,11 @@ function get(file) {
 
       https
         .get(url, res => {
-          res.pipe(f)
+          if (res.statusCode === 404) {
```

Comment:
here we only check for a 404, maybe we should handle it better, something like `res.statusCode > 400`